internal/authorization/converters.go

```diff
@@ -55,15 +62,20 @@ func (c IdentityConverter) TypeName() string {
 
 func (c IdentityConverter) Map(r *http.Request) []Permission {
 	id := chi.URLParam(r, "id")
+	var contextualTuples []openfga.Tuple
 
 	if id == "" {
-		// global is used in place of `*`, reason is OpenFGA has a special meaning for `*`
-		// see [public access](https://openfga.dev/docs/modeling/public-access)
-		id = "global"
+		id = GLOBAL_ACCESS_OBJECT_NAME
+		// Add contextual tuples to enable all users to list resources and give admins access to all resources
+		contextualTuples = append(
+			contextualTuples,
+			*openfga.NewTuple("user:*", CAN_VIEW, fmt.Sprintf("%s:%s", c.TypeName(), GLOBAL_ACCESS_OBJECT_NAME)),
```

Comment:
nitpick: why not saving the `Sprintf` result in a variable to reuse in these two tuples, instead of computing it twice? To push it a step further, we could have contants for these values since typename and global object name will remain the same

This nitpick applies to all other changed lines with code like this in the pr